docs/sources/style-guide/style-conventions/index.md

```diff
@@ -118,7 +119,7 @@ To focus a user's attention, Grafana Labs documentation includes notes, tips, ca
 
 The most common admonition is a note. A note provides additional information that the user should be aware of.
 
-The Markdown syntax for a note is `> **Note:**`. The note appears in the published output as follows:
+For example:
```

Comment:
Why are you removing the code syntax for creating an admonition (Line 121)?  I _**just**_ put this into the Writer's Handbook after one of my engineers asked me how we create admonitions and I realized that you cannot figure it out without going into the code.  And this repo is NOT public.

Reply:
Weird. I didn't intentionally remove _anything_.  I also created the working copy on my computer with a fresh repo and all-new clone of everything as of a few days ago. I can't think of any reason why your changes would not be here. Let me look into it further.

Reply:
That is weird!

Reply:
Perhaps I deleted it unintentionally? I don't know. Added it back, will push soon.

Reply:
This is fixed in latest file.